pkg/api/validation/validation.go

```diff
@@ -1875,6 +1900,7 @@ func ValidatePodTemplateSpec(spec *api.PodTemplateSpec, fldPath *field.Path) fie
 	allErrs := field.ErrorList{}
 	allErrs = append(allErrs, ValidateLabels(spec.Labels, fldPath.Child("labels"))...)
 	allErrs = append(allErrs, ValidateAnnotations(spec.Annotations, fldPath.Child("annotations"))...)
+	allErrs = append(allErrs, ValidatePodSpecificAnnotations(spec.Annotations, fldPath.Child("annotations"))...)
```

Comment:
I had fixed this already. Can you take a look at the push I did 10 seconds
ago ?

On Fri, Mar 4, 2016 at 12:19 PM, Tim Hockin <notifications@github.com>
wrote:

> In pkg/api/validation/validation.go
> <https://github.com/kubernetes/kubernetes/pull/20688#discussion_r55083329>
> :
>
> > @@ -1875,6 +1900,7 @@ func ValidatePodTemplateSpec(spec *api.PodTemplateSpec, fldPath *field.Path) fie
> >  	allErrs := field.ErrorList{}
> >  	allErrs = append(allErrs, ValidateLabels(spec.Labels, fldPath.Child("labels"))...)
> >  	allErrs = append(allErrs, ValidateAnnotations(spec.Annotations, fldPath.Child("annotations"))...)
> > +	allErrs = append(allErrs, ValidatePodSpecificAnnotations(spec.Annotations, fldPath.Child("annotations"))...)
>
> don't you need to call this from ValidatePod(), too ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/20688/files#r55083329>.
>